sycl/test/group_load_store_alignment_negative.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clangxx -fsycl %s -fsycl-device-only -Xclang -verify %s -Xclang -verify-ignore-unexpected=note,warning
+
+#include <sycl/sycl.hpp>
+
+using namespace sycl;
+using namespace sycl::ext::oneapi::experimental;
+
+// expected-error@* {{group_load requires a pointer if alignment property is used}}
+// expected-error@* {{group_load requires a pointer if alignment property is used}}
```

Comment:
Why do we expect two errors instead of one?

Reply:
Two errors are generated because there is a recursive call to group_load/group_store (with the same list of properties) - fallback to naive approach when some requirements are not satisfied,  so static_assert fires up twice, not sure how to easily avoid that.

Reply:
Can we split group_load/store functions to move static_assert outside for recursive function like this:

```c++
group_load () {
  static_assert();
  group_load_recursive(); // <- move current implementation there except static_assert
}
```

Reply:
Thanks, address that.